src/core/AutoRest/AutoRest.json

```diff
@@ -35,6 +35,9 @@
     },
     "AzureResourceSchema": {
       "type": "AzureResourceSchemaCodeGenerator, AutoRest.AzureResourceSchema"
+    },
+    "Go": {
```

Comment:
Does this generator support x-ms-odata, x-ms-pageable, x-ms-long-running-operations etc... If so i believe that this is not generic and should be named as "Azure.Go"

---
In reply to: [74860338](https://github.com/Azure/autorest/pull/1336#discussion_r74860338) [](ancestors = 74860338)